Makefile

```diff
@@ -110,4 +110,12 @@ deinit:
 	@echo "==> Deinitializing submodules"
 	@git submodule deinit --all -f
 
+install_misspell:
+	@echo "==> Installing mispell with go get"
+	@go get -u github.com/client9/misspell/cmd/misspell;
```

Comment:
I think this is also the first time we require a Go toolchain to be installed in order to work with this repository.

However, I feel okay about it since it's an optional tool and not a dependency of any of the "main" makefile targets. If we could arrange for this to run as a Travis-CI run or GitHub action in this repo then contributors to the repo could avoid using this tool directly if they don't already have Go installed and just let it happen automatically as a PR check.

In some other repositories we have a target just called `tools` which installs whatever tools are required for the current codebase. Maybe we could call this target `tools` instead for consistency with that. :man_shrugging: